utils/spaceship.uniqueKey/index.js

```diff
@@ -0,0 +1,9 @@
+function uniqueKey() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
```

Comment:
Why is there a `4` as default string?✋

Reply:
It was an intermediate separator, but I do not think I would need it.
i'ill remove it